client/src/main/scala/spatutorial/client/components/TerminalUserDeskRecs.scala

```diff
@@ -109,6 +113,72 @@ object TableTerminalDeskRecs {
     popover
   }).build
 
+  case class StaffMovementPopoverState(
+                                        hovered: Boolean = false,
+                                        reason: String = "",
+                                        date: String = "",
+                                        startTime: String = "",
+                                        endTime: String = "",
+                                        numberOfStaff: Int = 1
+                                      )
+
+  def staffMovementPopover(trigger: String, reason: String, startDate: SDate, endDate: SDate, bottom: String) = ReactComponentB[Unit]("staffMovementPopover")
+    .initialState_P((p) => {
+      StaffMovementPopoverState(
+        reason = reason,
+        date = f"${startDate.getDate}%02d/${startDate.getMonth}%02d/${startDate.getFullYear - 2000}%02d",
+        startTime = f"${startDate.getHours}%02d:${startDate.getMinutes}%02d",
+        endTime = f"${endDate.getHours}%02d:${endDate.getMinutes}%02d")
+    }).renderS((scope, state) => {
+    val popover = <.div(
+      ^.onMouseEnter ==> ((e: ReactEvent) => scope.modState(_.copy(hovered = true))),
+      //      ^.onMouseLeave ==> ((e: ReactEvent) => scope.modState(_.copy(hovered = false))),
+      if (state.hovered) {
+        PopoverWrapper(trigger = trigger, className = "staff-movement-popover", position = bottom)({
```

Comment:
oh, right! I wouldn't use a popover for this, it's really a modeless dialogy thing, right?